server/cmd/mmctl/commands/post.go

```diff
@@ -27,8 +28,8 @@ var PostCmd = &cobra.Command{
 var PostCreateCmd = &cobra.Command{
 	Use:     "create",
 	Short:   "Create a post",
-	Example: `  post create myteam:mychannel --message "some text for the post"`,
-	Args:    cobra.ExactArgs(1),
+	Example: `  post create myteam:mychannel "some text for the post"`,
```

Comment:
Could you please also add an example for a DM?

Reply:
Alright